# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/moz.build
# Commit: ce31f8095e7e
# Full Hash: ce31f8095e7e64e3e496ec4f42e17a3e578ce0cd
# Author: smolnar <smolnar@mozilla.com>
# Date: 2022-02-24 03:51:41
# Regressor Bug: 1751987
# File Overlap Count: 1
# Description:
#   Bug 1751987 - Fix python black lint failure. a=lint-fix CLOSED TREE
# ==============================================================================

diff -r 4f5567292c37 -r ce31f8095e7e widget/gtk/moz.build
--- a/widget/gtk/moz.build	Wed Feb 23 11:47:07 2022 +0000
+++ b/widget/gtk/moz.build	Wed Feb 23 14:03:29 2022 +0200
@@ -108,7 +108,7 @@
         "/media/mozva",
     ]
     USE_LIBS += [
-      "mozva",
+        "mozva",
     ]
 
 if CONFIG["MOZ_X11"]:
